### ISSUE TITLE
SCSS debug output show very little info in recent update

### ISSUE THREAD

**Issue by johanbrook** [fourseven/meteor-scss#86]
Hi! Great package! I've used it for a long while.

Recently (in the 3.0.0 release perhaps?) the debug output for the package started to show less info when an error occurred when building the SCSS files.

Currently, it looks like this:

```
=> Errors prevented startup:

   While building the application:
   native: Unexpected token E (compiling client/scss/base.scss)
   at Object.parse (native)
   at sourceHandler (<runJavaScript-54>:107:14)


=> Your application has errors. Waiting for file change.
```

Before, it looked like this:

```
=> Errors prevented startup:

   While building the application:
   client/scss/base.scss: Scss compiler error: undefined
   <path-to-app>/app/client/scss/_home.scss:102: invalid property name


=> Your application has errors. Waiting for file change.
```

Much more helpful with the specific file name, row number, and error.

I don't know if the change is in ' `node-sass`, `libsass`, or this package.

**Comment by fourseven** [comment]
It will be the API change in node-sass and me not updating this package to suit. I'll get a new release out soon which will include the newer node-sass as well.

**Comment by fourseven** [comment]
fixed in `v3.1.1`

**Comment by johanbrook** [comment]
Thanks!
